MatrixSDK/Background/Store/SyncResponseFileStore/SyncResponseFileStore.swift

```diff
@@ -0,0 +1,167 @@
+// 
+// Copyright 2020 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+@objcMembers
+/// Sync response storage in a file implementation.
+public class SyncResponseFileStore: NSObject {
+    
+    private enum SyncResponseFileStoreConstants {
+        static let folderName = "SyncResponse"
+        static let fileName = "syncResponse"
+        static let fileEncoding: String.Encoding = .utf8
+        static let fileOperationQueue: DispatchQueue = DispatchQueue(label: "SyncResponseFileStoreQueue")
```

Comment:
Why not in Queues enum like in the other place rather in a Constants block?

Note that having it static may be an issue for multi-account.

Reply:
You right. Removed static queue.